src/main/protobuf/game.proto

```diff
@@ -2,19 +2,237 @@ syntax = "proto3";
 
 package soc.protos;
 
-// All RPCs must have a request and response message, even if empty
 service CatanServer {
   rpc CreateGame (CreateGameRequest) returns (CreateGameResponse) {}
   rpc StartGame (StartGameRequest) returns (StartGameResponse) {}
-  // Subscribe to a game to recieve all its notifications
-  rpc Subscribe(SubscribeRequest) returns (stream GameUpdate) {}
-  rpc Move(MoveRequest) returns (MoveResponse) {}
+  rpc Subscribe(SubscribeRequest) returns (stream GameMessage) {}
```

Comment:
@grogdotcom This is really the file I want to focus on with you. The other stuff in this PR is just making sure I'm "backwards compatible" and can still simulate a game e2e with the client pinging to advance turns.